ios/browser/api/wallet/brave_wallet_service_factory.mm

```diff
@@ -0,0 +1,51 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#import "brave/ios/browser/api/wallet/brave_wallet_service_factory.h"
+
+#if !defined(__has_feature) || !__has_feature(objc_arc)
```

Comment:
This file doesn't use Objective-C or Objective-C++. Should be `.cc` like the others or remove above `ARC` stuff?

Reply:
Good point, we probably don't need obj-c++ for this